fe/fe-core/src/main/java/org/apache/doris/planner/SetOperationNode.java

```diff
@@ -144,6 +144,13 @@ public List<List<Expr>> getMaterializedConstExprLists() {
     @Override
     public void finalize(Analyzer analyzer) throws UserException {
         super.finalize(analyzer);
+        // the resultExprLists should be substituted by child's output smap
```

Comment:
if we has substitute `resultExprLists`, should we still need substitute when we generate `materializedResultExprLists` in next code block in finalize?

Reply:
modified, thanks.